constants/error-constants.js

```diff
@@ -123,6 +123,7 @@ const ERROR = {
     // Initialization
     CREATE_USER_MISSING_INFO: "Email and IDP are required to create a new user.",
     CREATE_USER_ORG_MISSING_INFO: "Organization ID is required to initialize the user's organization information.",
+    INVALID_ROLE_STUDY: "No permission the study.",
```

Comment:
Can we update this to make it clearer that it is a problem with the user's permissions and not a study permission problem? I would recommend "User does not have access to the study" but anything that indicates that it is a user permission problem would be fine

Reply:
Updated, thanks!